browser-extension/src/rateLaterHistory.js

```diff
@@ -162,12 +178,13 @@ const startHistoryCapture = async () =>
       displayHistoryVideoCount,
       displaySentVideoCount,
       displayFailedVideoCount,
+      importComplete,
       stopButton,
     } = addOverlay();
 
     let timeout;
 
-    const captureStep = async () => {
+    const captureStep = async (onEndOfHistory) => {
```

Comment:
Why did you choose to pass this as argument? This function already uses several variables defined in its parent scope.